package.json

```diff
@@ -534,15 +544,15 @@
         },
         {
           "command": "terraform.initCurrent",
-          "when": "config.terraform.languageServer.enable == true"
+          "when": "config.terraform.languageServer.enable == true && editorLangId == terraform"
         },
         {
           "command": "terraform.validate",
-          "when": "config.terraform.languageServer.enable == true"
+          "when": "config.terraform.languageServer.enable == true && editorLangId == terraform"
         },
         {
           "command": "terraform.plan",
-          "when": "config.terraform.languageServer.enable == true"
+          "when": "config.terraform.languageServer.enable == true && editorLangId == terraform"
```

Comment:
AFAICT the intention here was to hide the command when other, primarily non-Terraform languages are being edited.

Was it also intentional to hide it when `terraform-vars`, i.e. `*.tfvars` files are being edited? If not then I think the condition needs updating.